src/GitHub.VisualStudio/Commands/OpenFileInSolutionCommand.cs

```diff
@@ -0,0 +1,157 @@
+﻿using System;
+using System.ComponentModel.Composition;
+using GitHub.Services;
+using GitHub.VisualStudio;
+using GitHub.Services.Vssdk.Commands;
+using Microsoft.VisualStudio;
+using Microsoft.VisualStudio.Shell;
+using Microsoft.VisualStudio.Editor;
+using Microsoft.VisualStudio.Shell.Interop;
+using Microsoft.VisualStudio.TextManager.Interop;
+using Task = System.Threading.Tasks.Task;
+
+namespace GitHub.Commands
+{
+    [Export(typeof(IOpenFileInSolutionCommand))]
+    public class OpenFileInSolutionCommand : VsCommand, IOpenFileInSolutionCommand
```

Comment:
Need to come up with a better name for this command. It's responsible for navigating between a pull request file and the equivalent working directory file (and back again). Something like, `OpenPullRequestOrWorkingFile`?